flask_monitoringdashboard/__init__.py

```diff
@@ -19,15 +19,14 @@
 from flask_monitoringdashboard.core.config import Config
 from flask_monitoringdashboard.core.logger import log
 
-config = Config()
-
 
 def loc():
     """Get the current location of the project."""
     return os.path.abspath(os.path.dirname(__file__)) + '/'
 
 
-blueprint = Blueprint('dashboard', __name__, template_folder=loc() + 'templates')
+config = Config()
+blueprint = Blueprint(config.blueprint_name, __name__, template_folder=loc() + 'templates')
```

Comment:
eheh. as the number of users grows, we'll end up extracting all the hardcoded strings into configuration options :)